sky/skylet/events.py

```diff
@@ -128,7 +128,7 @@ def _run(self):
     def _stop_cluster(self, autostop_config):
         if (autostop_config.backend ==
                 cloud_vm_ray_backend.CloudVmRayBackend.NAME):
-            autostop_lib.set_is_autostopping()
+            autostop_lib.set_autostopping_indicator()
```

Comment:
I feel "indicator" still suggests it's a binary var/bool. How about `{set_}autostopping_started()`? Not feeling strongly.

Reply:
Good point! Renamed. Thanks!